fl4health/clients/basic_client.py

```diff
@@ -72,7 +72,7 @@ def __init__(
         # Attributes to be initialized in setup_client
         self.parameter_exchanger: ParameterExchanger
         self.model: nn.Module
-        self.optimizer: torch.optim.Optimizer
+        self.optimizers: Dict[str, torch.optim.Optimizer]
```

Comment:
Update optimizers attribute to be dict instead of a single optimizer which was necessary for the addition of ensemble models.